serverless/client/client.go

```diff
@@ -116,7 +116,7 @@ func NewProofBuilder(ctx context.Context, cp log.Checkpoint, h compact.HashFn, f
 // This function uses the passed-in function to retrieve tiles containing any log tree
 // nodes necessary to build the proof.
 func (pb *ProofBuilder) InclusionProof(ctx context.Context, index uint64) ([][]byte, error) {
-	nodes, err := merkle.CalcInclusionProofNodeAddresses(int64(pb.cp.Size), int64(index), int64(pb.cp.Size))
+	nodes, err := merkle.CalcInclusionProofNodeAddresses(int64(pb.cp.Size), int64(index))
```

Comment:
Needs `merkle.Rehash`, similarly to how it's in `ConsistencyProof`.

Reply:
aha, ta - this was unnecessary previously because of the logic which was previously inside `CalcInclusionProofNodeAddresses`?

Reply:
Right. Previously, it would return a different set of nodes if the 2 tree sizes are the same. Now it always returns non-ephemeral nodes and needs rehashing. With a future https://github.com/google/trillian/pull/2576 it will be possible to still workaround and get the ephemeral node, with a bit more conscious effort.

Reply:
👍